<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>513</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Notes app</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/5e83084845874-384x384.png</normaloff>:/icons/5e83084845874-384x384.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>391</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="layout1">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="main_list_button">
         <property name="text">
          <string>Main</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="archive_list_button">
         <property name="text">
          <string>Archive</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="sort_button">
         <property name="text">
          <string>Sort</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListView" name="listView_4"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>251</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_1" stretch="0">
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>35</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_13">
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>40</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_14">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item>
          <widget class="QLabel" name="categories">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Categories</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="personal_checkBox">
           <property name="text">
            <string>Personal</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="uni_checkBox">
           <property name="text">
            <string>University</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="work_checkBox">
           <property name="text">
            <string>Work</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="edit_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Edit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="renameButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Rename</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="archive_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Archive/Unarchive</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delete_button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="add_button">
         <property name="text">
          <string>Add note</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>737</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionArchive_Unarchive"/>
    <addaction name="actionEdit"/>
    <addaction name="actionDelete"/>
    <addaction name="actionShow_main_list"/>
    <addaction name="actionShow_archive_list"/>
    <addaction name="actionSort"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionShow_main_list"/>
   <addaction name="actionShow_archive_list"/>
   <addaction name="actionEdit"/>
   <addaction name="actionArchive_Unarchive"/>
   <addaction name="actionDelete"/>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/new-note.png</normaloff>:/icons/new-note.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionArchive_Unarchive">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/add-to-archive.png</normaloff>:/icons/add-to-archive.png</iconset>
   </property>
   <property name="text">
    <string>Archive/Unarchive</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/delete.png</normaloff>:/icons/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/edit.png</normaloff>:/icons/edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </action>
  <action name="actionShow_main_list">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/main list.png</normaloff>:/icons/main list.png</iconset>
   </property>
   <property name="text">
    <string>Show main list</string>
   </property>
  </action>
  <action name="actionShow_archive_list">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/archive.png</normaloff>:/icons/archive.png</iconset>
   </property>
   <property name="text">
    <string>Show archive list</string>
   </property>
  </action>
  <action name="actionSort">
   <property name="text">
    <string>Sort</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
